connectors/src/lib/models/intercom.ts

```diff
@@ -22,7 +23,8 @@ export class IntercomWorkspace extends Model<
   declare region: string;
 
   declare conversationsSlidingWindow: number;
-  declare shouldSyncAllConversations: boolean;
+  declare syncAllConversations: IntercomSyncAllConversationsStatus;
+  declare shouldSyncAllConversations: boolean; // @todo daph remove
```

Comment:
I initially created this field but I'd rather have a string with statuses over a boolean. 😅